templates/ContentGenerator/Instructor/UserDetail/set_row.html.ep

```diff
@@ -2,30 +2,39 @@
 %
 % # my ($set, $userSet, $mergedSet, $version) = @_;
 % my $setID   = $set->set_id;
+% my $isGateway = $set->assignment_type =~ /gateway/;
 % my $version = stash 'version';
 <tr>
-	<td class="text-center">
-		<label class="form-check-label">
-			<%= check_box $version ? "set.$setID,v$version.assignment" : "set.$setID.assignment" => 'assigned',
-				class => 'form-check-input', defined $mergedSet ? (checked => undef) : () =%>
-		</label>
-	</td>
-	<td class="text-center">
+	<th class="text-center" scope="row">
 		% if (defined $mergedSet) {
-			<b dir="ltr">
+			<span dir="ltr" class="px-2">
+				% if ($isGateway) {
+					<i class="icon fa-solid fa-list-check" data-alt="Test/Quiz" title="Test/Quiz"></i>
```

Comment:
This should be
```suggestion
					<i class="icon fa-solid fa-list-check" title="<%= maketext('Test/Quiz') %>"></i>
					<span class="visually-hidden"><%= maketext('Test/Quiz') %></span>
```
I want to eliminate the addition of these visually hidden elements with javascript, and just put them in directly.  I have a comment about doing so with a content generator method in `htdocs/js/system.js`, but I think even better is to just do it directly.  Note that the javascript adds the `sr-only-glyphicon` class which is pretty much identical in terms of css to Bootstrap's `visually-hidden` class.  Also, the text of this visually hidden element and the title should be translated.

Make the same change on line 26 below.